### ISSUE TITLE
tou.tv new french hulu clone

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#62]
```
Request for support for this site. 
```

Original issue reported on code.google.com by `grosnomb...@gmail.com` on 27 Jan 2010 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
Je pense que ca sera pas trop difficile, Le module de Theonion peut etre 
adapté.
```

Original comment by `stav...@gmail.com` on 31 Mar 2010 at 3:13

**Comment by GoogleCodeExporter** [comment]
```
See store-it.appspot.com/tou/tou.html   for details on how to extract the video 
by hand
```

Original comment by `stav...@gmail.com` on 31 Mar 2010 at 3:14

**Comment by GoogleCodeExporter** [comment]
```
Thanks, applied in r364.
```

Original comment by `monsieur...@gmail.com` on 3 Apr 2010 at 12:39
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
Is r364 included in the latest release (1.21)?
```

Original comment by `sgrego2...@gmail.com` on 15 Apr 2010 at 9:15
